x-pack/platform/plugins/shared/security/server/saved_objects/saved_objects_security_extension.ts

```diff
@@ -1698,58 +1697,6 @@ export class SavedObjectsSecurityExtension implements ISavedObjectsSecurityExten
     return this.auditLogger.includeSavedObjectNames;
   }
 
-  private setAccessControl({
```

Comment:
These two functions appeared to no longer be used, replaced by a utility within the extension.